src/main/scala/org/broadinstitute/dsde/agora/server/webservice/ApiService.scala

```diff
@@ -54,6 +54,15 @@ object ApiService extends LazyLogging with SprayJsonSupport with DefaultJsonProt
         complete(InternalServerError, AgoraException(e.getMessage, e.getCause, InternalServerError))
     }
   }
+
+  val rejectionHandler: RejectionHandler =
+    RejectionHandler.newBuilder().handle {
+      case MalformedRequestContentRejection(message, cause) => complete(BadRequest, message)
```

Comment:
This was only used in `ApiServiceSpec` but tests relied on this being handled in this way which is why I added it back to all routes.